src/server/pps/server/api_server.go

```diff
@@ -2419,7 +2419,7 @@ func (a *apiServer) stopAllJobsInPipeline(txnCtx *txncontext.TransactionContext,
 	jobInfo := &pps.JobInfo{}
 	sort := &col.Options{Target: col.SortByCreateRevision, Order: col.SortAscend}
 	err := a.jobs.ReadWrite(txnCtx.SqlTx).GetByIndex(ppsdb.JobsTerminalIndex, ppsdb.JobsTerminalKey(pipeline, false), jobInfo, sort, func(string) error {
-		return a.stopJob(txnCtx, jobInfo.Job, "pipeline updated")
+		return a.stopJob(txnCtx, jobInfo.Job, "pipeline stopped")
```

Comment:
Yeah - more details are needed but shouldn't it always say "pipeline stopped" followed by additional context?

Reply:
Message now looks like
```
all jobs killed because pipeline was updated for user zmajeed
```

or
```
all jobs killed because pipeline was stopped for user zmajeed
```

Reply:
Where do I get the host from?

Reply:
peer.FromContext will return a peer object containing the remote address: https://github.com/grpc/grpc-go/blob/v1.51.0/peer/peer.go#L48